packages/graphql-hooks/index.d.ts

```diff
@@ -172,6 +173,10 @@ interface UseQueryResult<
   ): Promise<UseClientRequestResult<ResponseData, TGraphQLError>>
 }
 
+interface UseSubscriptionOperation extends Operation {
```

Comment:
Perhaps we should rename this to `UseSubscriptionOptions`

Reply:
Leave `Operation` as is?

Reply:
One point in favor of *not* changing it to `useSubscriptionOptions` is that, unlike `useQuery`, the "options" (variables, operationName, client) are in the same object as `query`. I think, in hindsight, making `useSubscription` be more similar to `useQuery` with a signature of `query,options,callback` would have been better.

Reply:
Another point in favor of *not* changing it (at least not now), is that it's referred to as `Operation` or `operation` in a couple places (like `README`).

Reply:
@jackh726 Fair points 👏  - Let's leave as is for now.